static/scss/application.scss

```diff
@@ -183,3 +183,43 @@ $border-style: 1px solid $black;
     }
   }
 }
+
+.application-card-footer {
+  letter-spacing: 1.5px;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  text-align: right;
+
+  li {
+    display: inline-block;
+    margin-left: 30px;
+    vertical-align: top;
+  }
+
+  .collapse-link {
+    margin-left: 100px;
+  }
+
+  .view-statement {
+    color: #4e9eff;
+    font-weight: 300;
+  }
+
+  .btn-view-certificate {
+    background-color: #007ee5;
```

Comment:
don't use the hex value. we need to create a variable against that hex in `_variable.scss` and use that variable here.

Reply:
Updated.